### PR TITLE
making singlepage call to remote by default

### PR DESCRIPTION
**Pull request by prklm10**
Taking singlepage ss for app automate from mobile.
Adding `PERCY_DISABLE_REMOTE_UPLOADS` env variable to override above.
Adding `PERCY_ENABLE_DEV` for dev. Added it so we can test better with sdk.

### CODE REVIEW

File: percy/providers/app_automate.py [hunk 1]
```diff
@@ -36,13 +36,17 @@ def set_debug_url(self, session_details):
 
     def _get_tiles(self, **kwargs):
         fullpage_ss = kwargs.get('fullpage', False)
-        if not fullpage_ss:
+        if os.environ.get('PERCY_DISABLE_REMOTE_UPLOADS') == 'true':
             return super()._get_tiles(**kwargs)
+        screenshotType = 'singlepage'
+        if fullpage_ss:
+            screenshotType = 'fullpage'
```
Comment (ninadbstack) [inline percy/providers/app_automate.py:43]:
```suggestion
        screenshotType = 'fullpage' if fullpage_ss else 'singlepage'
```

Comment (prklm10) [reply percy/providers/app_automate.py]: done

File: tests/test_app_percy.py [hunk 2]
```diff
@@ -15,7 +16,7 @@
 from percy.metadata import AndroidMetadata
 from tests.mocks.mock_methods import android_capabilities, ios_capabilities
 
-
+@patch.dict(os.environ, {"PERCY_DISABLE_REMOTE_UPLOADS": "true"})
```
Comment (ninadbstack) [inline tests/test_app_percy.py:19]: What tests cover the cases when this is not set or set to false ? shouldnt we have an e2e test for that as well ?

Comment (prklm10) [reply tests/test_app_percy.py]: added e2e tests for when disable upload is not given.

**Review by pankaj443 — APPROVED** [review]

**Pull request merged**